scripts/migration/remove_invalid_social_entries.py

```diff
@@ -0,0 +1,43 @@
+"""
```

Comment:
This still needs to be made into a proper migration. It should be located in `osf/migrations` and follow our conventions for custom Django migrations (see other custom migrations in that folder for examples).